ESMF/Apps/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
-ecbuild_add_executable(
-        TARGET SyntheticApp.x
-        SOURCES SyntheticApp.F90 Synthetic_Driver.F90 Provider_GridComp.F90 Reciever_GridComp.F90 UFS_TestingCap.F90
-        LIBS gftl-shared MAPL MAPL.NUOPC
-        INCLUDES ${esma_include}/MAPL
-)
+add_executable(SyntheticApp.x
```

Comment:
@rmontuoro Are any of the `Apps` used for your UFS work? If not then they can be deleted as they were just used for development testing

Reply:
@WilliamJamieson - `Apps` are not being used for UFS development work. I'm fine with deleting the subdirectory if you agree.

Reply:
@rmontuoro I am fine with doing this as well.

Reply:
Commit f8c2208 removed test `Apps`